shaka-lab-gateway-client/macos/README.md

```diff
@@ -0,0 +1,57 @@
+# Shaka Lab Gateway Client
+
+The Shaka Lab Gateway Client package allows Active Directory users from Shaka
+Lab Gateway to log in.
+
+This is the macOS package.
+
+**NOTE**: Exactly one Linux device on your lab network must install the
+[`shaka-lab-gateway`](https://github.com/shaka-project/shaka-lab/tree/main/shaka-lab-gateway#readme)
+package.  Devices with the client package will connect to that gateway.
+
+## Pre-requisites
+
+ - Get Homebrew: [https://brew.sh/#install](https://brew.sh/#install)
+ - You must grant full disk access to Terminal before proceeding.  Without this
```

Comment:
There's also the "brew doctor" command that gives you a list of specific folders you should change the ownership of (among other things).
Or would that not help for our case?

Reply:
No, I don't think that helps.  My understanding is that by default, you can't make changes to arbitrary files from Terminal until you do this, and this permission is needed for Homebrew or one of the scripts.

I can't recall the details, and it's always possible I was wrong.  I just got another Mac for the lab, though, so I'll leave this step in the docs for now, but I'll try without this step on a fresh device and see what breaks.  If nothing breaks, I'll remove this from the docs.  If something breaks, I'll add more details to the docs.

Reply:
I double-checked on a fresh install, and I found that gateway-client works fine without this.  In fact, it is only recommended-settings that requires Terminal to have full disk access, to change OS settings from the CLI.

So I'm removing this from the gateway-client doc, but leaving it in recommended-settings.